### PR TITLE
Sent email ui

### PR DESCRIPTION
**Pull request by jay-meister**

### CODE REVIEW

**Comment by jay-meister** [comment]
Getting the following error:

```
interfaces/sanctuary.js:7
  7:     (f: (b: b) => c): (g: (a: a) => b) => (a: a) => c
                               ^^^^^^^^^^^ Library type error:
  7:     (f: (b: b) => c): (g: (a: a) => b) => (a: a) => c
                               ^^^^^^^^^^^ function type. Callable signature not found in
 38:     const newState = compose(dissoc('custom_emails'), dissoc('email_sent'))(state)
                                                                                 ^^^^^ object literal. See: src/admin/redux/modules/email/reducer.js:38

interfaces/sanctuary.js:7
  7:     (f: (b: b) => c): (g: (a: a) => b) => (a: a) => c
                               ^^^^^^^^^^^ Library type error:
  7:     (f: (b: b) => c): (g: (a: a) => b) => (a: a) => c
                               ^^^^^^^^^^^ function type. Callable signature not found in
 38:     const newState = compose(dissoc('custom_emails'), dissoc('email_sent'))(state)
                                                                                 ^^^^^ object type. See: src/admin/redux/modules/email/reducer.js:38

src/admin/redux/modules/email/reducer.js:38
 38:     const newState = compose(dissoc('custom_emails'), dissoc('email_sent'))(state)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ function call
 53:         return { ...newState, custom_emails: { members: payload.results }}
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ object literal. Expected object instead of
 53:         return { ...newState, custom_emails: { members: payload.results }}
                         ^^^^^^^^ statics of function type
```

**Comment by rjmk** [comment]
For ease, I defined every Sanctuary fn as unary. So compose will need to be used as `compose(f)(g)` as opposed to `compose(f, g)`

File: api/services/email_mailgun.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ module.exports = {
     if (process.env.NODE_ENV === 'testing') {
       return callback(undefined, 'Email sent')
     }
+    //callback(null, ['jmurphy.web@mail.com', 'ivan@foundersandcoders.com'])//TODO delete line
```
Comment (rjmk) [inline api/services/email_mailgun.js]: Shall we delete this?

File: src/admin/containers/email.js [hunk 2]
```diff
@@ -35,10 +36,18 @@ const Email = (
...
+      ? <EmailNotification email_sent={email_sent}/>
+      : (custom_emails
+            ? <CustomEmailForm submit={submit_custom_email} members={custom_emails.members} email_sent={email_sent} />
+            : keys(emails).length > 0 && email_list({ emails, submit_email, ...list_props })
+        )
+    }
   </div>
 
+const EmailNotification = ({email_sent}) => email_sent === 'success'
+  ? <h3 className='sent-email-notification'>The emails have been sent</h3>
```
Comment (rjmk) [inline src/admin/containers/email.js]: Let's move the ternary to the smallest scope possible -- ie inside the `h3`

File: src/admin/containers/email.js [hunk 3]
```diff
@@ -35,10 +36,18 @@ const Email = (
     >
       { map(send_button, zip(email_ids, [sub, news, remind, custom])) }
     </form>
-    {custom_emails ? <CustomEmailForm submit={submit_custom_email} members={custom_emails.members}/>
-  : keys(emails).length > 0 && email_list({ emails, submit_email, ...list_props })}
+    {email_sent
+      ? <EmailNotification email_sent={email_sent}/>
```
Comment (rjmk) [inline src/admin/containers/email.js:40]: Is this nested ternary removed in later work?

File: src/admin/redux/modules/email/reducer.js [hunk 4]
```diff
@@ -37,7 +37,7 @@ const initialState = { emails: { } }
 
 const reducer : Reducer<State, Action>
   = (state = initialState, { type, payload }) => {
-    const newState = dissoc('custom_emails', state)
+    const newState = compose(dissoc('custom_emails'))(dissoc('email_sent'))(state)
```
Comment (rjmk) [inline src/admin/redux/modules/email/reducer.js]: Probably best to use `omit` here

File: src/admin/redux/modules/email/reducer.js [hunk 5]
```diff
@@ -75,6 +74,10 @@ const Email = content => (
   }
 )
 
+const email_response = state => resBody => resBody
+  ? { ...state, email_sent: 'success' }
```
Comment (rjmk) [inline src/admin/redux/modules/email/reducer.js]: Again, let's move the ternary further in to the value of `email_sent`

File: src/styles/pages/_index.scss [hunk 6]
```diff
@@ -545,6 +545,11 @@ div.gbp {
   color: black;
 }
 
+.sent-email-notification {
+  font-size: 25px;
```
Comment (rjmk) [inline src/styles/pages/_index.scss]: `px` should really be avoided

File: src/admin/containers/email.js (hunk 3 above)
Comment (jay-meister) [inline src/admin/containers/email.js:40]: At the moment it only gets more nested 😟  in later work

But we can change that (today?) in the bounced emails branch so will leave this in for now.

File: src/styles/pages/_index.scss [hunk 7]
```diff
@@ -546,7 +546,7 @@ div.gbp {
 }
 
 .sent-email-notification {
-  font-size: 25px;
+  font-size: 1.5em;
   margin: 50px 0 0 7px;
```
Comment (rjmk) [inline src/styles/pages/_index.scss]: `px` here too!

Comment (jay-meister) [inline src/styles/pages/_index.scss]: We shouldn't use px at all?

Comment (rjmk) [inline src/styles/pages/_index.scss]: Outside of known sizes for the screen / format, I don't think so. What happens if someone visits with a very different number of pixels on their screen than you?

I know that CSS pixels are not exactly device pixels, but this still seems like a problem to me

File: src/admin/containers/email.js (hunk 3 above)
Comment (jay-meister) [inline src/admin/containers/email.js:40]: Done in bounced-emails branch

**Pull request merged**